theme/global/global.ts

```diff
@@ -34,16 +35,21 @@ export const globalCss = defineGlobalStyles({
     },
   },
   '.mode__popup': {
-    'html,body, #app, .radix-themes': {
-      maxHeight: '100vh',
+    '&, body': {
+      height: '100%',
+      width: '100%',
+      //TODO use monorepo token
```

Comment:
I need to get [this PR merged](https://github.com/leather-io/mono/pull/304) to trigger a release of `tokens` so I can consume this new value properly. 

I [added it yesterday](https://github.com/leather-io/mono/pull/303) but just as I was merging realised I made a mistake using `chore:` in the commit message so it won't release it